optimade/grammar/v0.10.1.lark

```diff
@@ -49,7 +49,7 @@ set_op_rhs: HAS ( [ OPERATOR ] value
                  | ONLY value_list )
 
 // Note: support for [ OPERATOR ] is OPTIONAL
-length_op_rhs: LENGTH [ OPERATOR ] value
+length_op_rhs: LENGTH [ OPERATOR ] signed_int
```

Comment:
This makes sense.
I was thinking it would be cool if one could put another property name here, however, I guess a client would simply get the int from the property in question before creating the query - so no matter.
We should check this is in line with the spec though, and we're not supporting a stricter filter grammar than what is set out by the spec.

Reply:
I think this is fine, if we loosen the requirement to a "number" then the query will fail internally. The spec just says `value` which can be an int or a float.